lib/covenants/namestate.js

```diff
@@ -209,7 +209,7 @@ class NameState extends bio.Struct {
     if (this.isClaimable(height, network))
       return false;
 
-    // If we haven't been renewed in a year, start over.
+    // If we haven't been renewed in two years, start over.
     if (height >= this.renewal + network.names.renewalWindow)
```

Comment:
Was initial plan to have it expire in a year ?

Reply:
haha yeah I guess. There's a bunch of outdated docs and comments we're still finding since mainnet:

https://github.com/handshake-org/hsd/pull/556/files
https://github.com/handshake-org/handshake-web/pull/7
https://github.com/handshake-org/handshake-web/pull/4
https://github.com/handshake-org/handshake-web/pull/4